src/deadline/client/api/_submit_job_bundle.py

```diff
@@ -134,6 +135,11 @@ def create_job_from_job_bundle(
                 ProgressReport as a parameter, which can be used for projecting remaining time, as in done in the CLI.
     """
 
+    if not submitter_name:
```

Comment:
How come the switch from required with default to optional with hand-coded default?

Reply:
This fixes a bug from a previous change I made to this code. Previously we added the ability to customize the submitter name as a CLI argument (`--submitter-name`) and the intention was the if one isn't provided, the default value would be used. The defaults were set as function parameter defaults. What I noticed was actually happening was that the if the CLI argument is not present, this function gets called with `None` as the `submitter_name` value and the default doesn't get applied. This change makes it so `None` values are replaced by the submitter name default.